### ISSUE TITLE
[Election-Nominations] Name for this project

### ISSUE THREAD

**Issue by ChargingBulle** [LibreBlockGame/GameToBeRenamed#1]
# Propose project names here

This issue is for collecting potential names for the fork.
The election timeline will be announced shortly (that's my responsibility at the moment).

You can nominate as many names as you want.
We want to be Minetest-compatible but not Minetest-associated. This project has it's own identity independant from Minecraft, Minetest, Infiniminer etc.

**Comment by numberZero** [comment]
V-core for the engine (V stands for “voxel” of course). For the whole thing... Voxelation?

**Comment by ChargingBulle** [comment]
OpenCraft

**Comment by DrAzraelTod** [comment]
BlockGameThing

**Comment by numberZero** [comment]
> OpenCraft

[Too late](https://sourceforge.net/projects/opencraft/), fortunately. My suggestion may not be very creative or even good, but this one sounds too much like “poor man’s MineCraft” (or even WarCraft, out of context), even more than “Minetest” does.

> BlockGameThing

That assumes this will be a game, not an “engine” or whatever. But #2 is not answered yet...

BTW, found on the forum: Voxelia

**Comment by opensofias** [comment]
Voxelverse

**Comment by opensofias** [comment]
Voxblox

**Comment by opensofias** [comment]
Voxcraft

**Comment by opensofias** [comment]
Worxels (world pixels/voxels/elements)

**Comment by opensofias** [comment]
Pickaxe

**Comment by DrAzraelTod** [comment]
"Mein Block"

I'll show myself out…

**Comment by DrAzraelTod** [comment]
> > BlockGameThing
> 
> That assumes this will be a game, not an “engine” or whatever
> 

Isn't a engine some game thing though?

**Comment by ChargingBulle** [comment]
LibreCraft

**Comment by ChargingBulle** [comment]
OpenMC

but that's perhaps a bit on the nose

**Comment by ChargingBulle** [comment]
In 2016 the official project made this poll: https://www.strawpoll.me/32807
(mentioned here just for the sake of completeness)

**Comment by ChargingBulle** [comment]
FreeMine
OpenMine / OpenMiner
VoxBox
The OpenMine Engine/TOME
(from above mentioned 2016 poll)

**Comment by DrAzraelTod** [comment]
> OpenMC
> 
> but that's perhaps a bit on the nose

it worked for OpenMicrowave uh… I mean… [OpenMW](https://openmw.org/en/)

**Comment by numberZero** [comment]
> LibreCraft

Sorry but LibreOffice is there already.

> FreeMine

[Freeminer](https://github.com/freeminer/freeminer) is there already. That name is a bit better though as it has some sense beyond “free alternative to Minecraft”.

> OpenMine

As in “open mind”? Dunno.

What’s about a name without “Open”, “Free” or “Libre” prefix, for reasons I [stated](https://github.com/LibreBlockGame/GameToBeRenamed/issues/1#issuecomment-1006946130) 15 hours ago? It doesn’t even need to mention voxels or crafting or whatever... not any more than “Firefox” mentions HTML, for example.

**Comment by ChargingBulle** [comment]
> What’s about a name without “Open”, “Free” or “Libre” prefix, for reasons I https://github.com/LibreBlockGame/GameToBeRenamed/issues/1#issuecomment-1006946130 15 hours ago? It doesn’t even need to mention voxels or crafting or whatever... not any more than “Firefox” mentions HTML, for example.

Yeah that makes sense. "Firefox" is a great name. It's not TheMozHTMLRenderingEngine but simply Firefox.
